src/sql/src/plan/query.rs

```diff
@@ -5586,15 +5590,50 @@ impl Visit<'_, Aug> for WindowFuncCollector {
     }
 }
 
-/// Specifies how long a query will live. This impacts whether the query is
+/// Specifies how long a query will live.
+///
+/// Currently, it affects whether SHOW commands are allowed.
```

Comment:
Opinion: this sentence and the following explainer paragraph should be removed. Instead, the various impl methods should have explainer text about their intentions and what they should and shouldn't allow.

Reply:
I removed them and added comments on the helper functions.